pkg/app/mysql.go

```diff
@@ -42,9 +42,19 @@ type MysqlDB struct {
 	chart     helm.ChartInfo
 }
 
+type MysqlDBOptions func(*MysqlDB)
+
+func MysqlDBWithChart(chart map[string]string) MysqlDBOptions {
```

Comment:
should this be?

```suggestion
func MysqlDBWithValues(values map[string]string) MysqlDBOptions {
```

Reply:
or I think better wouldbe `MysqlDBHelmValues`, because we are just creating helm values for options. but it's just TILI.

Reply:
perfect.

Reply:
Fixed.